test/service/RuntimeService.test.ts

```diff
@@ -0,0 +1,62 @@
+import { expect } from 'chai';
+import { it } from 'mocha';
+import { LoggerFactory, LogType, RuntimeService } from '../../src';
+const logger = LoggerFactory.getLogger(LogType.Silent);
+const service = new RuntimeService(logger);
+describe('RuntimeService', async () => {
+    it('exec when valid', async () => {
```

Comment:
does this test pass on windows?

Reply:
cc @gimre-xymcity

Reply:
Jumped over to windows and now all tests are running on windows. Afair,  Travis's windows vm doesn't support docker. Keen to migrate to github actions if have the time (maybe actions works better, need to revisit)